composer.json

```diff
@@ -14,11 +14,11 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0"
+        "php": ">=5.3"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.*",
-        "satooshi/php-coveralls": "0.*"
+        "phpunit/phpunit": "^4.0|^5.0",
```

Comment:
My concern is less the tests and more that running the tests and the code in the same version is a "good idea" (tm).
I think the better step is to make both the library and tests require 5.6 (and above) and set a new major version, even if it's just for that. Can you adjust that, including the master branch alias? Then I'll tag it as the next major version, to make sure it'll be froward compatible.